CromwellRefdiskManifestCreator/src/test/java/org/broadinstitute/manifestcreator/CromwellRefdiskManifestCreatorAppTest.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.manifestcreator;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.broadinstitute.manifestcreator.CromwellRefdiskManifestCreatorApp.Arguments;
+import org.broadinstitute.manifestcreator.model.ReferenceDiskManifest;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.*;
+import static org.hamcrest.Matchers.*;
+
+public class CromwellRefdiskManifestCreatorAppTest {
+
+  @Test
+  public void testManifestCreation() throws InterruptedException, IOException {
```

Comment:
TOL does this run with our CI?

Reply:
No. Actually this project is completely detached from Cromwell's sbt project. It just resides in the cromwell root directory, but it has to be built separately.
I can reimplemented it in scala, and make it a module of cromwell project, but I'm not sure it's worth doing.